packages/e2e-tests/import-queries/__tests__/import-queries.spec.ts

```diff
@@ -162,4 +162,11 @@ describe.runIf(!isBuild)('ssrLoadModule', () => {
 		const result = await ssrLoadDummy('?raw&svelte&type=style');
 		await expect(result).toMatchFileSnapshot(snapshotFilename('ssr-style'));
 	});
+	test('?inline&svelte&type=style&lang.css', async () => {
+		// Preload Dummy.svelte first so its CSS is processed in the module graph, otherwise loading
```

Comment:
wait, wouldn't this also affect real use if the inline style is explicitly asked for without the .svelte being processed first? The solution here would be to compile the file if cache is empty instead of just bailing.

Reply:
If the virtual styles (inline or not) is requested before the main module is compiled, it'll not work like before. If we want to allow accessing the virtual styles directly and automatically compile the main module, I think that's a separate feature. Was also discussed in https://github.com/sveltejs/vite-plugin-svelte/issues/341

Reply:
makes sense to put it in a separarate PR. :+1: